CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## Master
+
+* Adds initializing android sdk from using `Instabug.start` instead of inside the application class
```

Comment:
I think we should add more details here. Are users updating to this version going to need to change anything?